infrastructure/env/common/tfvars.json

```diff
@@ -15,5 +15,11 @@
   "azurerm_apim_eventhub_rule": "apim-eventhub-rule",
   "azurerm_kubernetes_admin_username": "k8s-admin",
   "azurerm_kubernetes_admin_ssh_publickey_file": "k8s_admin_key.pub",
-  "azurerm_dns_main_zone": "io.italia.it"
+  "default_sender_email": "no-reply@io.italia.it",
+  "azurerm_dns_main_zone": "io.italia.it",
+  "azurerm_dns_main_zone_spf1": "v=spf1 include:musv.com ~all",
```

Comment:
I don't understand what this setting is supposed to mean.

Reply:
this setups a dns zone (server) to handle the io.italia.it subdomain. then it adds a txt record for spf1 and two cnames for dkim. the default_sender_email param is the default email address we use to send any email from d.c.